### PR TITLE
Jellyfin port

### PR DESCRIPTION
**Pull request by cvium**
Cherry picked some of the compatible upstream fixes and fixed Nuget deps.

Rebranded sln.

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Force-pushed back to commit 26905e18b47aa661e6b012138f198fd355ad3d52 to facilitate this.

File: Jellyfin.Plugin.Reports/Api/Data/ReportExport.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ﻿using System.Linq;
 using System.Text;
-using JellyfinReports.Api.Common;
-using JellyfinReports.Api.Data;
-using JellyfinReports.Api.Model;
+using Jellyfin.Plugin.Reports.Api.Model;
```
Comment (Bond-009) [inline Jellyfin.Plugin.Reports/Api/Data/ReportExport.cs]: alphabetical order

File: Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-<PropertyGroup> <TargetFrameworks>netstandard2.0;</TargetFrameworks> <AssemblyVersion>10.1.0.0</AssemblyVersion> <FileVersion>10.1.0.0</FileVersion> <RootNamespace>JellyfinReports</RootNamespace> </PropertyGroup>
+<PropertyGroup> <TargetFrameworks>netstandard2.0;</TargetFrameworks> <AssemblyVersion>10.1.0.0</AssemblyVersion> <FileVersion>10.1.0.0</FileVersion> <RootNamespace>Jellyfin.Plugin.Reports</RootNamespace> </PropertyGroup>
```
Comment (Bond-009) [inline Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj]: Proper indentation

File: Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <TargetFrameworks>netstandard2.0;</TargetFrameworks>
```
Comment (Bond-009) [inline Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj]:
```suggestion
  <TargetFramework>netstandard2.0</TargetFramework>
```
This and indentation are the last nitpicks, I swear

**Pull request merged**